src/client/pages/SignIn.tsx

```diff
@@ -175,14 +142,13 @@ export const SignIn = ({
 	recaptchaSiteKey,
 	isReauthenticate = false,
 	shortRequestId,
-	defaultView = 'passcode',
-	currentView = defaultView,
 	usePasscodeSignIn = false,
+	hideSocialButtons = false,
 }: SignInProps) => {
 	// status of the OTP checkbox
-	const [selectedView, setSelectedView] = React.useState(
+	const [selectedView] = React.useState(
```

Comment:
Does this still need to be a React state if its readonly?

Reply:
Ah yes, good point! I'll make it static.